DiffusionPreprocessing/DiffPreprocPipeline.sh

```diff
@@ -93,11 +93,10 @@
 
 # Set up this script such that if any command exits with a non-zero value, the 
 # script itself exits and does not attempt any further processing.
```

Comment:
Ok, I hate to suggest further changes on this PR, but for the scripts that had these comments preceding `set -e`
```
# Set up this script such that if any command exits with a non-zero value, the 
# script itself exits and does not attempt any further processing.
```
now that `set -e` is removed, shouldn't those comment lines be removed as well?  Otherwise, those comments don't seem to apply to anything.

Reply:
Yes, you are right. It is a pain though ... I'll get to it.

Reply:
Hmmm ... I found just two files with those comments.

Reply:
By default, debug.shlib does effectively the same thing as set -e, but with more verbosity, so that comment still applies, as long as it is near the debug sourcing.  I don't care if these comments exist.

Reply:
I weeded them out. I'd rather not be putting them in everywhere.